src/mcp_atlassian/server.py

```diff
@@ -2362,6 +2371,37 @@ def format_comment(comment: Any) -> dict[str, Any]:
                     )
                 ]
 
+        elif name == "jira_get_link_types" and ctx and ctx.jira:
+            if not ctx or not ctx.jira:
+                raise ValueError("Jira is not configured.")
```

Comment:
This check `if not ctx or not ctx.jira:` is redundant because the `elif` condition on above line already ensures that both `ctx` and `ctx.jira` are valid (truthy) before entering this block. This check can be safely removed.

Reply:
I updated this by myself as this is minor